implementing-nosto/implement-search/implement-search-using-code-editor/implementing-search-page.md

```diff
@@ -438,7 +460,24 @@ init({
 })
 ```
 
-Once fallback is enabled, if the search request fails to retrieve data, the search functionality will be temporarily disabled for a short period, and the user will be redirected to the same page they are currently on.
+Once fallback is enabled, if the search request fails to retrieve data, the search functionality will be temporarily disabled for a short period, and the native content Nosto has overridden will be restored.
```

Comment:
I think the documentation using "native" and "original" content interchangeably could cause confusion. We can stick to a single word, if possible